fp.sql

```diff
@@ -0,0 +1,88 @@
+WITH gather_survey_data AS (
+    SELECT
+        site_short AS survey_site_short,
+        SUM(
+            CASE
+                WHEN contact_id IS NOT NULL THEN 1
+                ELSE 0
+            END
+        ) AS ps_survey_scholarship_denom,
+        SUM(
+            CASE
+                WHEN i_am_able_to_receive_my_scholarship_funds_from_college_track IN ('StronglyAgree', 'Strongly Agree', 'Agree') THEN 1
```

Comment:
@jvolan123 separate Q. Should we account for 'StronglyAgree' entries for other survey KPIs?